src/arch/riscv/insts/vector.cc

```diff
@@ -575,51 +614,53 @@ VlFFTrimVlMicroOp::branchTarget(ThreadContext *tc) const
     uint32_t new_vl = calcVl();
 
     pc_ptr->as<PCState>().vl(new_vl);
-    return std::unique_ptr<PCStateBase>{pc_ptr};
+    return std::unique_ptr<PCStateBase>{ pc_ptr };
 }
 
 std::string
 VlFFTrimVlMicroOp::generateDisassembly(Addr pc,
-    const loader::SymbolTable *symtab) const
+                                       const loader::SymbolTable *symtab) const
 {
     std::stringstream ss;
     ss << mnemonic << " vl";
     return ss.str();
 }
 
-std::string VlSegMacroInst::generateDisassembly(Addr pc,
-        const loader::SymbolTable *symtab) const
+std::string
+VlSegMacroInst::generateDisassembly(Addr pc,
+                                    const loader::SymbolTable *symtab) const
 {
     std::stringstream ss;
-    ss << mnemonic << ' ' << registerName(destRegIdx(0)) << ", " <<
-        '(' << registerName(srcRegIdx(0)) << ')' <<
-        ", " << registerName(srcRegIdx(1));
+    ss << mnemonic << ' ' << registerName(destRegIdx(0)) << ", " << '('
+       << registerName(srcRegIdx(0)) << ')' << ", "
+       << registerName(srcRegIdx(1));
     if (!machInst.vm)
         ss << ", v0.t";
     return ss.str();
 }
 
-std::string VlSegMicroInst::generateDisassembly(Addr pc,
-        const loader::SymbolTable *symtab) const
+std::string
+VlSegMicroInst::generateDisassembly(Addr pc,
+                                    const loader::SymbolTable *symtab) const
 {
     std::stringstream ss;
-    ss << mnemonic << ' ' << registerName(destRegIdx(0)) << ", " <<
-        '(' << registerName(srcRegIdx(0)) << ')' <<
-        ", "<< registerName(srcRegIdx(1));
+    ss << mnemonic << ' ' << registerName(destRegIdx(0)) << ", " << '('
+       << registerName(srcRegIdx(0)) << ')' << ", "
+       << registerName(srcRegIdx(1));
     if (microIdx != 0 || machInst.vtype8.vma == 0 || machInst.vtype8.vta == 0)
         ss << ", " << registerName(srcRegIdx(2));
     if (!machInst.vm)
         ss << ", v0.t";
     return ss.str();
 }
 
-VlSegDeIntrlvMicroInst::VlSegDeIntrlvMicroInst(ExtMachInst extMachInst, uint32_t _micro_vl,
-                        uint32_t _dstReg, uint32_t _numSrcs,
-                        uint32_t _microIdx, uint32_t _numMicroops,
-                        uint32_t _field, uint32_t _vlen, uint32_t _sizeOfElement)
-    : VectorArithMicroInst("vlseg_deintrlv_micro", extMachInst,
-                            SimdAddOp, 0, 0),
-        vlen(_vlen)
+VlSegDeIntrlvMicroInst::VlSegDeIntrlvMicroInst(
+    ExtMachInst extMachInst, uint32_t _micro_vl, uint32_t _dstReg,
+    uint32_t _numSrcs, uint32_t _microIdx, uint32_t _numMicroops,
+    uint32_t _field, uint32_t _vlen, uint32_t _sizeOfElement)
+    : VectorArithMicroInst("vlseg_deintrlv_micro", extMachInst, SimdAddOp, 0,
+                           0),
+      vlen(_vlen)
```

Comment:
Can we clang-format off in `VlSegDeIntrlvMicroInst::VlSegDeIntrlvMicroInst` section?